zeppelin-server/pom.xml

```diff
@@ -197,6 +197,22 @@
       <version>2.2.1</version>
     </dependency>
 
+    <dependency>
+      <groupId>com.sun.jersey</groupId>
+      <artifactId>jersey-servlet</artifactId>
+      <version>1.13</version>
+    </dependency>
+
+    <dependency>
+      <groupId>javax.ws.rs</groupId>
+      <artifactId>javax.ws.rs-api</artifactId>
+      <version>2.0-m10</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.scala-lang</groupId>
+      <artifactId>scala-library</artifactId>
```

Comment:
Good catch, need to check if we do not specify it already somewhere.
Although it looks like a bit out of scope of this PR.